build.sbt

```diff
@@ -1958,24 +1980,22 @@ lazy val designer = (project in file("designer/server"))
         CopyOptions.apply(overwrite = true, preserveLastModified = true, preserveExecutable = false)
       )
     },
+    prepareDesignerTests             := {
```

Comment:
Why did you merge dependencies for both test scopes? Doesn't it slow down test on CI?